src/date-picker/index.tsx

```diff
@@ -173,8 +173,9 @@ const DatePicker = React.forwardRef(
                 {...focusVisible}
                 tabIndex={0}
                 className={styles.calendar}
-                role="application"
+                role="dialog"
                 aria-describedby={calendarDescriptionId}
+                aria-label={'Calendar dialog'}
```

Comment:
Can we take ariaLabelledBy, ariaDescribedby from the form-field-context, same as in the drp?

Reply:
It is a good point. But ariaLabelledBy, ariaDescribedby from the form-field-context should apply to the trigger not the calendar dialog? 
We can maybe use arialabel from datepicker, just like in DRP https://github.com/cloudscape-design/components/blob/main/src/date-range-picker/dropdown.tsx#L148 ?